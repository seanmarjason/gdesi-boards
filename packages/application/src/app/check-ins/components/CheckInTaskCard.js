import Card from '@mui/material/Card';
import CardContent from '@mui/material/CardContent';
import Link from '@mui/material/Link';
import Typography from '@mui/material/Typography';


export default function CheckInTaskCard({task, index}) {
    return (  
        <Card sx={{ minWidth: 275, maxWidth: 400 }} key={index}>
        <CardContent>
            <Typography gutterBottom sx={{ fontSize: 14 }}>
                <Link href={`/tasks/${task.id}`} underline="none" >
                    {task.id}
                </Link>
            </Typography>
            <Typography variant="h5" component="div">
            {task.title}
            </Typography>
            <Typography sx={{ color: 'text.secondary', mb: 1.5 }}>
            {task.type}
            </Typography>
        </CardContent>
        </Card>
    )
}
