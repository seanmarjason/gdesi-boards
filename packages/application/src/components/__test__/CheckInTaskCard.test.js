import { render } from '@testing-library/react'
import CheckInTaskCard from '../CheckInTaskCard';
  
const task = {
    id: '',
    title: '',
    type: ''
}
 
it('CheckInTaskCard renders correctly', () => {
  const { container } = render(<CheckInTaskCard task={task} index={1} />)
  expect(container).toMatchSnapshot()
})
