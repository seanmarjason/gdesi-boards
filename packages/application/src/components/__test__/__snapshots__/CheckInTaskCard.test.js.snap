// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckInTaskCard renders correctly 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1qgr1xq-MuiPaper-root-MuiCard-root"
    style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
  >
    <div
      class="MuiCardContent-root css-1lt5qva-MuiCardContent-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1wdikoc-MuiTypography-root"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-18rqtra-MuiTypography-root-MuiLink-root"
          href="/boards/tasks/"
        />
      </p>
      <div
        class="MuiTypography-root MuiTypography-h5 css-16xl4zq-MuiTypography-root"
      />
      <p
        class="MuiTypography-root MuiTypography-body1 css-17v1mfu-MuiTypography-root"
      />
    </div>
  </div>
</div>
`;
